html/semantics/forms/the-textarea-element/textarea-placeholder-manaul.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML Test: textarea - placeholder attribute</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="http://www.w3.org/TR/html5/forms.html#the-textarea-element">
```

Comment:
We point to the most recent spec: <https://html.spec.whatwg.org/multipage/#attr-textarea-placeholder>.